tests/unit/test_utils.py

```diff
@@ -100,3 +102,10 @@ def test_get_packit_version():
 )
 def test_ensure_str(inp, exp):
     assert ensure_str(inp) == exp
+
+
+@pytest.mark.parametrize(
+    "to,from_,exp", (("/", "/", "."), ("/a", "/a/b", ".."), ("/a", "/c", "../a"))
+)
+def test_relative_to(to, from_, exp):
+    assert os.path.relpath(to, from_) == exp
```

Comment:
I don't mind having it here, I'm just wondering why we test library function :-)

Reply:
it's probably a leftover from my code when I wanted to make sure it does what I expect it to do

Reply:
Yes, as Tomas said, should I remove it?

Reply:
As I said, I don't mind, keep it there if you want.